test/OpenTelemetry.Tests/Shared/TestActivityExportProcessor.cs

```diff
@@ -15,20 +15,20 @@
 // </copyright>
 
 using System.Collections.Generic;
+using System.Diagnostics;
 
 namespace OpenTelemetry.Tests
 {
-    internal class TestExportProcessor<T> : BaseExportProcessor<T>
-        where T : class
+    internal class TestActivityExportProcessor : SimpleActivityExportProcessor
```

Comment:
This one you might want to leave as it was. Nothing type-specific about it and it was kind of a nice helper for writing tests over Activity or LogRecord.

Reply:
I had to change this because this is used in the tests in ExportProcessorTest.cs which assert that activity should not be exported if Actvity.Recorded is false. The check used to happen in the OnEnd of BaseExportProcessor<T> before. With this approach, that check has been moved to the OnEnd of SimpleActivityExportProcessor and BatchActvitiyExportProcessor.